src/rdf.rs

```diff
@@ -0,0 +1,231 @@
+        dataset.clone(),
+        rdf::TYPE,
+        dcat::DATASET_CLASS,
+        GraphName::DefaultGraph,
+    ))?;
+    Ok(store)
+}
+
+pub fn add_five_star_annotation(store: &Store) -> Result<BlankNode, StorageError> {
+    let five_star_annotation_node = BlankNode::default();
+    store.insert(
+        Quad::new(
+            five_star_annotation_node.as_ref(),
+            rdf::TYPE,
+            dqv::QUALITY_ANNOTATION_CLASS,
+            GraphName::DefaultGraph,
+        )
+        .as_ref(),
+    )?;
+
+    Ok(five_star_annotation_node)
+}
+
+pub fn add_derived_from(
+    quality_annotation: NamedOrBlankNodeRef,
+    derived_from: NamedOrBlankNodeRef,
+    store: &Store,
+) -> Result<(), StorageError> {
+    store.insert(
+        Quad::new(
+            quality_annotation,
+            prov::WAS_DERIVED_FROM,
+            derived_from,
+            GraphName::DefaultGraph,
+        )
+        .as_ref(),
+    )?;
+    Ok(())
+}
+
+// Add quality measurement to metric store
+pub fn add_quality_measurement(
+    metric: NamedNodeRef,
+    computed_on: NamedOrBlankNodeRef,
+    value: bool,
+    store: &Store,
+) -> Result<BlankNode, StorageError> {
+    let measurement = BlankNode::default();
+    let value_term = Term::Literal(Literal::new_typed_literal(
+        format!("{}", value),
+        xsd::BOOLEAN,
+    ));
+
+    store.insert(
+        Quad::new(
+            measurement.as_ref(),
+            rdf::TYPE,
+            dqv::QUALITY_MEASUREMENT_CLASS,
+            GraphName::DefaultGraph,
+        )
+        .as_ref(),
+    )?;
+    store.insert(
+        Quad::new(
+            measurement.as_ref(),
+            dqv::IS_MEASUREMENT_OF,
+            metric,
+            GraphName::DefaultGraph,
+        )
+        .as_ref(),
+    )?;
+    store.insert(
+        Quad::new(
+            measurement.as_ref(),
+            dqv::COMPUTED_ON,
+            computed_on,
+            GraphName::DefaultGraph,
+        )
+        .as_ref(),
+    )?;
+    store.insert(
+        Quad::new(
+            measurement.as_ref(),
+            dqv::VALUE,
+            value_term,
+            GraphName::DefaultGraph,
+        )
+        .as_ref(),
+    )?;
+    store.insert(
+        Quad::new(
+            computed_on,
+            dqv::HAS_QUALITY_MEASUREMENT,
+            measurement.as_ref(),
+            GraphName::DefaultGraph,
+        )
+        .as_ref(),
+    )?;
+
+    Ok(measurement)
+}
+
+// Dump graph as turtle string
+pub fn dump_graph_as_turtle(store: &Store) -> Result<Vec<u8>, SerializerError> {
+    let mut buffer = Vec::new();
+    store.dump_graph(&mut buffer, GraphFormat::Turtle, GraphNameRef::DefaultGraph)?;
+    Ok(buffer)
+}
+
+// Check if format is RDF
+pub fn is_rdf_format(format: &str) -> bool {
+    match format.to_lowercase().as_str() {
+        "rdf" => true,
+        "turtle" => true,
+        "ntriples" => true,
+        "n3" => true,
+        "nq" => true,
+        "json-ld" => true,
+        "jsonld" => true,
```

Comment:
Litt enklere å se at alle disse gir `true` når man bruker `|` (or)
```suggestion
        "rdf" | "turtle" | "ntriples" | "n3" | "nq" | "json-ld" | "jsonld" => true,
```